indico/htdocs/sass/partials/_tables.scss

```diff
@@ -30,6 +36,27 @@ table.i-table {
         font-weight: bold;
         text-align: left;
         background-color: #EAEAEA;
+
+        &:not(.sorter-false) {
```

Comment:
Yes, it is internally used by the jquery.tablesorter. However I think I can be more explicit here (and use aforementioned selector), since I am using `data-sorter` in the session list